packages/shared/src/components/post/PostSummaryButton.tsx

```diff
@@ -22,7 +22,7 @@ export function PostSummaryButton({
 
   return (
     <>
-      <div className="absolute inset-0 bg-gradient-to-t from-background-default from-60% to-overlay-primary-pepper" />
+      <div className="absolute inset-0 bg-gradient-to-t from-raw-pepper-90 from-60% to-overlay-primary-pepper" />
```

Comment:
Did you check with design?
Sometimes its accidental they picked a non named version.

Ideally i want to fully remove raw colors soon 🙏

Reply:
They mentioned they really don't know what to call the new color, and it might be a one-time thing. To avoid raw colors, I instead used the theme-based approach through JS.

Reply:
Ah I see :) 
Just something to keep in mind I guess (as they easily creep in and stay forever)